var/spack/repos/builtin/packages/baurmc/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import shutil
+
+from spack import *
+
+
+class Baurmc(AutotoolsPackage):
+    """Baur's Monte Carlo package for simulating W+ gamma production at hadron
+    colliders provides full electroweak calculation upto O(alpha) and contains
+    higher order QCD corrections as well."""
+
+    homepage = "https://twiki.cern.ch/twiki/bin/view/Sandbox/BaurMCInteface"
+    url      = "http://lcgpackages.web.cern.ch/lcgpackages/tarFiles/sources/MCGeneratorsTarFiles/baurmc-1.0-src.tgz"
+
+    tags = ['hep']
+
+    version('1.0', sha256='de5027ed2e66028bed890760bee9d869e1e330ac7f7112ee5cb25868cea5c35b')
+
+    def do_stage(self, mirror_only=False):
+        # the tarball extracts to an intermediate directory -
+        # move everything to the proper source dir
+        super(Baurmc, self).do_stage(mirror_only)
+        dn = os.listdir(self.stage.source_path)[0]
+        for fn in os.listdir(join_path(self.stage.source_path, dn)):
+            shutil.move(join_path(self.stage.source_path, dn, fn),
+                        join_path(self.stage.source_path, fn))
+        shutil.rmtree(join_path(self.stage.source_path, dn))
```

Comment:
Can't you set the `configure_path` property instead of moving everything?

Reply:
Do you have a link to docs or an example? (Grepping the builtin spack repo for `configure_path` comes up empty)

Reply:
Apologies, I meant `configure_directory`:
https://github.com/spack/spack/blob/62abbeaf6f9e6aaa8f34baa2ef1ee57139577ea8/lib/spack/spack/build_systems/autotools.py#L285-L291

Reply:
Ah, I see , thanks for the pointer.

However, the intermediate directory is names after the version (extracts to `spack-src/1.0/configure`) so I'm not sure hardcoding that is an improvmement.

Reply:
I'm not sure I understand the issue. Wouldn't:
```python
@property 
def configure_directory(self): 
    return os.path.join(self.stage.source_path, str(self.spec.version))
```
be fine?

Reply:
Yes, that works, thanks!